src/plugins/intel_cpu/src/thread_pool_imp.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <common/dnnl_thread.hpp>
+#include <iostream>
+#include <oneapi/dnnl/dnnl_threadpool.hpp>
+#include <oneapi/dnnl/dnnl_threadpool_iface.hpp>
+
+#include "cpu_parallel.hpp"
+#include "openvino/core/parallel.hpp"
+
+namespace ov::intel_cpu {
+
+class ThreadPool : public dnnl::threadpool_interop::threadpool_iface {
+private:
+    std::shared_ptr<CpuParallel> m_cpu_parallel = nullptr;
+    ov::intel_cpu::TbbPartitioner m_partitioner = ov::intel_cpu::TbbPartitioner::STATIC;
+    size_t m_multiplier = 32;
```

Comment:
The multiplier is stored in the m_cpu_parallel. It's more logical and safe implement the same method in `CpuParallel` amd call it instead, correctly binding the behavior of the `CpuParallel` and `ThreadPool`.

Reply:
updated